### ISSUE TITLE
Documentation is missing information about AJV custom errors 

### ISSUE THREAD

**Issue by nikooiko** [strongloop/loopback-next#5954]
I was following [this guide](https://loopback.io/doc/en/lb4/Parsing-requests.html) in order to add custom AJV error messages to a model definition and I noticed that they don't work. (also found a similar [discussion](https://stackoverflow.com/questions/62334308/loopback-4-custom-error-messages-on-models/62949155) at stack overflow)

@hacksparrow informed me that the [documentation](https://github.com/strongloop/loopback-next/blob/master/docs/site/Parsing-requests.md) is missing some information about the body-parser options that must be defined in order to enable the extra functionality for custom errors.

Missing Piece of code:
```js
this.bind(RestBindings.REQUEST_BODY_PARSER_OPTIONS).to({
  validation: {
    ajvKeywords: true,
    ajvErrors: true
  }
});
```

**Comment by dhmlau** [comment]
PR in progress https://github.com/strongloop/loopback-next/issues/5937 by @hacksparrow

**Comment by dhmlau** [comment]
Closing as resolved via PR #5937.
